integrations/access/servicenow/config.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicenow
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/gravitational/trace"
+	"github.com/pelletier/go-toml"
+
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/integrations/access/common"
+	"github.com/gravitational/teleport/integrations/lib"
+)
+
+// Config stores the full configuration for the teleport-servicenow plugin to run.
+type Config struct {
+	common.BaseConfig
+	ClientConfig
+	Servicenow common.GenericAPIConfig
+}
+
+// LoadServicenowConfig reads the config file, initializes a new ServicenowConfig struct object, and returns it.
+// Optionally returns an error if the file is not readable, or if file format is invalid.
+func LoadServicenowConfig(filepath string) (*Config, error) {
```

Comment:
This code doesn't seem to be used. Was it used for tests?
Do we configure our plugins using toml or is it used by `teleport-plugins` repo